src/mlvl_wrapper.rs

```diff
@@ -63,6 +63,29 @@ impl<'r, 'mlvl, 'cursor, 'list> MlvlArea<'r, 'mlvl, 'cursor, 'list>
         x.kind.as_mrea_mut().unwrap()
     }
 
+    pub fn toggle_memory_relay(&mut self, mem_relay_id: u32, active: u8)
```

Comment:
Toggle is the wrong word. Your function is a setter for the default state of the relay.

Reply:
set_memory_relay_active then

Reply:
Done